src/js/utils.js

```diff
@@ -57,7 +57,9 @@ export function iconPad(props) {
 
   let style = '';
   if (height && theme?.text?.[height]?.height) {
-    const lineHeight = parseMetricToNum(theme.text[height].height);
+    const unit = theme.text[height].height.match(/[a-z]+$/)?.[0] || 'px';
```

Comment:
Why do we need both the regex and the `||` for px? I would think the regex would match both `rem` and `px` -- also I would think this would match any a-z chars such as `em`,`ft`, even `zebra`.

Should we make the regex a bit more targeted to the specific units which are valid/we support?

Reply:
Made adjustments in latest commit.